app/models/employee.rb

```diff
@@ -12,6 +12,6 @@ class Employee < ApplicationRecord
   has_many :rewards, through: :orders
 
   def points
-    received_kudos.count - rewards.sum(:price).to_i
+    received_kudos.count - rewards.sum(:price).to_i + number_of_available_kudos
```

Comment:
In my understanding, `number_of_available_kudos` should be an employee's attribute holding current amount of points (But as far as I remember, couple of sprints ago the requirement was to not implement such thing) - we would need Oskar's opinion on this one.

Reply:
this one has changed recently - we added a task to sprint 2 with this requirement. Nonetheless, it all comes down to having a DB column on employees' table that holds an amount of kudos a user could give. As you might have this task locked for a few days (we're working on a fix) I'm attaching a screenshot of the task's desc and AC from our backoffice.
<img width="1303" alt="Screenshot 2022-03-11 at 11 13 06" src="https://user-images.githubusercontent.com/5034850/157847454-bcb4e4f2-eb11-4177-822a-925587aa5949.png">